src/runServer.ts

```diff
@@ -41,14 +41,17 @@ function start(retries: number, config: ConnectOptions, databaseDir: string, isW
   });
 
   function errorCatch(err: Error) {
+    function fatalError(e: Error) {
+      logger.error("Fatal Error:", e);
+      process.exit(1);
+    }
     if (retries > 0) {
       logger.warn(err.message);
       retries--;
-      augurNodeController.shutdown();
+      augurNodeController.shutdown().catch(fatalError);
```

Comment:
I don't believe `shutdown()` will ever reject since the entire thing takes place in try with no rethrow

Reply:
This `.catch(fatalError)` added to satisfy new eslint rules for dangling promises